03_launch_mgmt_cluster.sh

```diff
@@ -238,8 +222,8 @@ EOF
     "${BMOPATH}/tools/deploy.sh" false true "${IRONIC_TLS_SETUP}" "${IRONIC_BASIC_AUTH}" true
 
     # Restore original files
-    mv "${BMOPATH}/ironic-deployment/ironic/ironic.yaml.orig" "${BMOPATH}/ironic-deployment/ironic/ironic.yaml"
-    mv "${BMOPATH}/ironic-deployment/keepalived/keepalived_patch.yaml.orig" "${BMOPATH}/ironic-deployment/keepalived/keepalived_patch.yaml"
+    mv "${BMOPATH}/ironic-deployment/default/kustomization.yaml.orig" "${BMOPATH}/ironic-deployment/default/kustomization.yaml"
+    mv "${BMOPATH}/ironic-deployment/keepalived/kustomization.yaml.orig" "${BMOPATH}/ironic-deployment/keepalived/kustomization.yaml"
```

Comment:
Trying to understand why we are moving root kustomization.yaml instead of a specific file?

Reply:
It is because the [make targets](https://github.com/metal3-io/baremetal-operator/blob/main/Makefile#L156-L179) we use now (instead of `sed` on the specific files). They use `kustomize edit set image`, which changes the `kustomization.yaml` by adding something like this:
```yaml
images:
- name: name-of-image-in-manifest
  newName: new-image-name
  newTag: x.y
```